mammoth/opts.py

```diff
@@ -537,10 +350,12 @@ def model_opts(parser):
     group.add(
         '--transformer_ff', '-transformer_ff', type=int, default=2048, help='Size of hidden transformer feed-forward'
     )
+    # TODO is this actually in use?
```

Comment:
It is passed around in the code. Not sure if functional, though.

Reply:
leaving is as is for now, ideally #56 would remove that